### PR TITLE
D8 nid 419

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: nidirect_contacts/src/Plugin/Field/FieldFormatter/GMapsLazyLoadFormatter.php [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+
+    $formatter_settings = $this->getSettings();
+    $google_provider = \Drupal::service('plugin.manager.geolocation.mapprovider')->getMapProvider('google_maps');
+    $google_settings = \Drupal::config('geolocation_google_maps.settings');
```
Comment (DuttonMa) [inline nidirect_contacts/src/Plugin/Field/FieldFormatter/GMapsLazyLoadFormatter.php:156]: Could Dependency Injection be used here ?

File: nidirect_contacts/nidirect_contacts.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+gmaps_lazy_load:
+  js:
+    js/gmaps-lazy-load.js: {}
+  dependencies:
+    - core/jquery
+    - core/drupalSettings
```
Comment (DuttonMa) [inline nidirect_contacts/nidirect_contacts.libraries.yml:6]: should this depend on geolocation as well ?

File: nidirect_contacts/src/Plugin/Field/FieldFormatter/GMapsLazyLoadFormatter.php (hunk 1 above)
Comment (omahm) [reply nidirect_contacts/src/Plugin/Field/FieldFormatter/GMapsLazyLoadFormatter.php:156]: True, my bad

File: nidirect_contacts/nidirect_contacts.libraries.yml (hunk 2 above)
Comment (omahm) [reply nidirect_contacts/nidirect_contacts.libraries.yml:6]: The formatter includes the required geolocation libraries

**Pull request merged**